go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/hyperledger/fabric
 
 go 1.14
 
+// https://github.com/golang/go/issues/34610
+replace golang.org/x/sys => golang.org/x/sys v0.0.0-20190920190810-ef0ce1748380
```

Comment:
Out of curiosity, why do the replace rather than pin it the version in the `require` block

Reply:
I'm still learning the tao of modules so it may not be a best practice.

Putting the version in the require block gets us the right version but it doesn't prevent a `go get -u` from updating it automatically. So if, for example, someone wanted to update grpc, the version of x/sys would also get bumped because that package depends on a slightly newer version. By using a `replace` directive, we unilaterally pick the version.

I think if the change impacted CI or development on Darwin, I'm not sure I would bother with a replace because our regular builds would fail. Since it's just for Windows, I wanted to highlight it.

I can go either way.

Reply:
The question was really just for my curiosity and understanding. Like you I am just trying to understand everything with modules as well. While it's a step in the right direction, it doesn't totally behave live other software package managers I am used to (such as npm).

It makes sense to use the replace directive to prevent a transient dependency (such as gRPC) from updating the package to something that doesn't work for us.

Thanks for the clarification.